eclipse-plugin/sapl-eclipse-feature/feature.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2017-2024 Dominic Heutelbeck (dominic@heutelbeck.com)
+
+    SPDX-License-Identifier: Apache-2.0
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<feature id="sapl-eclipse-feature"
+	label="SAPL Feature "
+	version="3.0.0.qualifier">
+<!--	<plugin-->
```

Comment:
Delete commented code?

Reply:
Removed now. It was left over from an earlier state of the eclipse plugin before the restructuring where I converted sapl-lang and sapl-ide from eclipse-plugins to regular osgi-bundles.